htdocs/holiday/card.php

```diff
@@ -792,7 +794,7 @@
 				}
 
 				// Calculcate number of days consummed
-				$nbopenedday = num_open_day($object->date_debut_gmt, $object->date_fin_gmt, 0, 1, $object->halfday);
+				$nbopenedday = $object->nb_open_day;
```

Comment:
If you remove the call here to num_open_day(), we should retreive the call somewhere else. But i can't find it anywhere in your new code...